bundles/org.openhab.binding.sony/src/main/java/org/openhab/binding/sony/internal/net/Slf4LoggingAdapter.java

```diff
@@ -0,0 +1,49 @@
+/**
+ * Copyright (c) 2010-2020 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.sony.internal.net;
+
+import java.util.Objects;
+import java.util.logging.LogRecord;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.eclipse.jdt.annotation.Nullable;
+import org.slf4j.Logger;
+
+/**
+ * Logging adapter to use for Slf4j
+ *
+ * @author Tim Roberts - Initial Contribution
+ */
+@NonNullByDefault
+class Slf4LoggingAdapter extends java.util.logging.Logger {
+    /** The logger */
+    private final Logger logger;
+
+    /**
+     * Creates the logging adapter from the given logger
+     *
+     * @param logger a non-null logger to use
+     */
+    protected Slf4LoggingAdapter(final Logger logger) {
+        super("jersey", null);
+        Objects.requireNonNull(logger, "logger cannot be null");
+        this.logger = logger;
+    }
+
+    @Override
+    public void log(@Nullable final LogRecord record) {
+        if (record != null) {
+            logger.debug("{}", record.getMessage());
+        }
+    }
```

Comment:
isn't this what the jul-to-slfj4 bridge is for?

Reply:
I have absolutely no idea what that is - please explain more?

Reply:
The jul-to-slf4j library acts as a bridge between the java.util.logging (jul) classes and slf4j. It automatically forwards all logs coming from jul to slf4j without any work from the developer. By just including the library you can continue to use the jul logging framework but have the logging destination controlled by slf4j. I don't know for sure, but it seemed that was what you were going for in this class.

Reply:
I can't find any instances of this being used in the code base - do you have an example I can follow?

Reply:
Not sure what was intended here as comment. Except that `"{}"` isn't generally not recommended and at least provide some context. If the message would be empty it will just print nothing.

Reply:
@Hilbrand - @cpmeister was correct in that this class only provides an adapter between java logging and SLF4 (so basically I'm taking the whole logging message from java logging and reposting it to SLF4).  It's a simple adapter class and what @cpmeister noted that there is a project (jul-to-slfj4) has this class built in.  My question was if anyone uses that and if so, point me to that implementation and I can copy it.  If I'm the only one using it - I'd rather stick with my simple class rather than complicate thing by bringing in a whole new package.  However, if others are using this - than I'd be fine to convert to it..

Reply:
I don't see jul-to-slf4j used anywhere in openhab so you might be the first. jul-to-slf4j is released by the same folks as the slf4j-api, thus they have the same licenses. Given that openhab already uses slf4j-api I don't think there would be a problem with including it yourself, but doesn't sound like you want to go through the hassle.